lib/ansible/modules/web_infrastructure/sentry_project.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Mikhail Naletov <okgolove@markeloff.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: sentry_project
+short_description: Manage Sentry project
+description:
+    - Manage the state of a project in Sentry.
+version_added: "2.8"
+
+options:
+  api_token:
+    description:
+      - API token generated in Sentry. The token has to have an acess for manipulating projects.
+    required: true
+
+  organization:
+    description:
+      - Organization containing a project.
+    required: true
+
+  project_name:
+    description:
+      - Project name you want to work with.
+    required: true
+
+  project_slug:
+    description:
+      - Slug name. Slug name must be uniq (even after project deletion you can't create a project with same slug name).
+    required: true
+
+  state:
+    description:
+      - The state of project.
+    required: true
+    choices: [ "present", "absent" ]
+
+  team:
+    description:
+      - A team you are in and which has an access for project manipulating.
+
+  url:
+    description:
+      - Sentry URL with scheme.
+    required: true
+
+author: "Mikhail Naletov (@okgolove)"
+'''
+
+EXAMPLES = '''
+# Create project "backend" in "example" organization and "senior" team
+- sentry_project:
+    api_token: 1234567890abcdwxyz
+    organization: example
+    project_name: backend
+    project_slug: backend
+    state: present
+    team: senior
+    url: https://sentry.example.com
+'''
+
+RETURN = ''' # '''
+
+import json
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils._text import to_native
+
+
+def main():
+    arg_spec = dict(
+        api_token=dict(required=True),
+        organization=dict(required=True),
+        project_name=dict(required=True),
+        project_slug=dict(required=True),
+        state=dict(required=True, choices=['present', 'absent']),
+        team=dict(required=True),
+        url=dict(required=True)
+    )
+
+    module = AnsibleModule(argument_spec=arg_spec, supports_check_mode=False)
```

Comment:
Supporting check mode would be nice... seeing you already do checks to see if a project exists, check mode isn't _that_ much of a stretch.